lib/OpenQA/WebAPI/Controller/API/V1/JobSettings.pm

```diff
@@ -8,11 +8,18 @@ sub jobs ($self) {
     my $validation = $self->validation;
     $validation->required('key')->like(qr/^[\w\*]+$/);
     $validation->required('list_value')->like(qr/^\w+$/);
+    $validation->optional('results')->like(qr/^\w+$/);
+    $validation->optional('list_groupids')->like(qr/^[\w\*]+$/);
```

Comment:
For other queries we support "group_glob" and "not_group_globs". Introduing "list_groupids" here looks like a new, inconsistent approach. Also see the second suggestion in https://progress.opensuse.org/issues/131279#Suggestions How about extending the current group_glob support into here?

As alternative, potentially even simpler, see https://progress.opensuse.org/issues/131279#Suggestions number 3 stating `"squads" could be mapped into openQA for example with special job settings, e.g. QE Core ensures to trigger all their tests with _SQUAD='QE Core' and then be able to filter by that`

Reply:
I dont see a straight way to invoke group_glob inside the lib/OpenQA/WebAPI/Controller/API/V1/JobSettings.pm. That is because the variables are tight inside [_compose_job_overview_search_args](https://github.com/os-autoinst/openQA/blob/69e8e802aadb9b4d330fb30e0701c8f6a84f5945/lib/OpenQA/WebAPI/Plugin/Helpers.pm#L394C5-L394C38).

Reply:
If the code cannot be reused it would at least make sense to keep the design/usage consistent from a user perspective. (And I think this is also mainly what @okurz was asking for.)